app/policies/dashboard_policy.rb

```diff
@@ -5,7 +5,14 @@
 class DashboardPolicy < ApplicationPolicy
   sig { returns(T::Boolean) }
   def show?
-    administrator? || collection_creator? || user.deposits.any? || user.reviews_collections.any?
+    # If/when we add the designed "first time user" workflow, we would want to prevent
+    # users who don't have any deposits from going right to the dashboard. As of
+    # Feb 2021, we haven't had time to do that so we allow any user who is a depositor
+    # in any collection.
+    # administrator? || collection_creator? || user.deposits.any? ||
+    #   user.reviews_collections.any? || user.manages_collections.any?
```

Comment:
Did you want to leave this here?

Reply:
Yes, that's a breadcrumb for when we have the first deposit workflow implemented.

Reply:
Intentionally crumby code, cool.